src/app/shared/entry.ts

```diff
@@ -196,4 +197,55 @@ export abstract class Entry implements OnInit, OnDestroy, AfterViewInit {
     }
     return selectedVersion;
   }
+
+  // Selects a tab of index tabIndex
+  selectTab(tabIndex: number) {
+    this.entryTabs.tabs[tabIndex].active = true;
+  }
+
+  // Sorts two entries by last modified, and then verified
+  entryVersionSorting(a: any, b: any): number {
```

Comment:
Not entirely sure if this is what's wanted.  Should double-check with Denis just in case.  Need an example (might as well put it in a test too).

Reply:
Ya I thought it was sort by last modified, and then verified to deal with tie breakers. That is what the spec said.

Test is a good idea.

Reply:
I kind of interpreted as sort by last modified, then take all the verified ones (while retaining order), and throw it to the top.

Reply:
I could be wrong, but they should be the same, assuming a stable sort

Reply:
An example would be great.  Right now, it looks like:

- Sort by date.  If same date, put verified above the other one.

I thought it'd be:

- Sort by verified (verified ones on top).  If same verified status, put newer ones above the other ones.